packages/build-utils/src/fs/glob.ts

```diff
@@ -64,5 +72,20 @@ export default async function glob(
     }
   }
 
+  // Add empty directory entries
+  for (const relativePath of dirs) {
+    if (dirsWithEntries.has(relativePath)) continue;
```

Comment:
**suggestion (non-blocking):** Should this look for the dirname of the path or am I misreading this?
 
```suggestion
    if (dirsWithEntries.has(path.dirname(relativePath))) continue;
```

Reply:
We already did dirname in the loop above that populates this set, so this is correct.